holoviews/tests/util/test_transform.py

```diff
@@ -4,8 +4,13 @@
 import pickle
 import warnings
 
+import holoviews as hv
+from io import StringIO
+hv.extension('bokeh')
```

Comment:
`hv.extension` should not be run in the test suite, and it isn't needed for your test to pass.

Reply:
Sounds good, I remove it.